packages/commonwealth/client/scripts/views/components/FeedDiscovery/FeedDiscovery.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import {Helmet} from "react-helmet";
+
+export default function FeedDiscovery() {
+
+  if (document.location.pathname.endsWith('/discussions')) {
```

Comment:
If we call this component from the DiscussionsPage.tsx page we can:
1. Remove this if statement, since we know that it is called on the discussions page
2. Pass in the state of the filters as props to remove all the url parsing and clean up this component

Reply:
agree

Reply:
I moved this to a page specific component so this is no longer an issue!